content/docs/rfcs/16/README.md

```diff
@@ -117,23 +117,6 @@ none
 
 Refer to the [Waku Relay specification](https://github.com/vacp2p/specs/blob/master/specs/waku/v2/waku-relay.md) for more information on the relaying of messages.
 
-### Types
-
-The following structured types are defined for use on the Relay API:
-
-#### WakuRelayMessage
-
-`WakuRelayMessage` is an `Object` containing the following fields:
-
-| Field | Type | Inclusion | Description |
-| ----: | :---: | :---: | ----------- |
-| `payload` | `String` | mandatory | The payload being relayed as a hex encoded data string |
-| `contentTopic` | `String` | optional | Message content topic for optional content-based filtering |
-| `timestamp` | `Number` | optional | The time at which the message is generated by its sender. This field holds the Unix epoch time in nanoseconds as a 64-bits integer value.  |
-| `ephemeral` | `Boolean` | optional | This flag indicates the transient nature of the message. |
-
- > **_NOTE:_** `WakuRelayMessage` maps directly to a [`WakuMessage`](#wakumessage), except that the latter contains an explicit message `version`. For `WakuRelay` purposes, the versioning is handled by the API.
-
 ### `post_waku_v2_relay_v1_message`
```

Comment:
This is a breaking change for this API method (also the Private API methods, but these are presumable very seldomly used). Since the Relay API is used by several applications/platforms, wouldn't it make sense to maintain backwards compatibility with this API version (at least in the implementation) and introduce `post_waku_v2_relay_v2_message` (note `v2` indicates new API version). The versioning as part of the method signature was introduced to prevent backwards incompatibilities.
We could also deprecate this function, but it may be a good idea to maintain backwards compatibility for one release cycle (i.e. until nwaku `v0.16.0`).